docs/index.rst

```diff
@@ -358,9 +358,10 @@ Priority sampling
 Priority sampling consists in deciding if a trace will be kept by using a `priority` attribute that will be propagated
 for distributed traces. Its value gives indication to the Agent and to the backend on how important the trace is.
 
-- 0: Don't keep the trace.
+- -1: The user asked not to keep the trace.
```

Comment:
Same suggestions for other tracers. What if we provide a module with priority values? something like http://www.django-rest-framework.org/api-guide/status-codes/#status-codes

```python
# NOTE: all names are just an examples
from ddtrace.ext import priority

# somewhere in the code
ctx.sampling_priority = priority.KEEP_TRACE
```